packages/webdriver/src/request/factory.ts

```diff
@@ -17,9 +16,7 @@ export default class RequestFactory {
         isHubCommand: boolean = false
     ): Promise<WebDriverRequest> {
         if (!EnvRequestLib) {
-            EnvRequestLib = process?.versions?.node
-                ? (await import('./node.js')).default
-                : (await import('./browser.js')).default
+            EnvRequestLib = (await import('./request.js')).default
```

Comment:
I think we can refactor this as we don't have a need for this factory anymore.

Reply:
We should also get rid of the `node:url` import in favor for https://nodejs.org/docs/latest/api/globals.html#url which is more cross platform compatible